<!-- -->
<launch>

  <!-- send table.xml to param server -->
  <arg name="pose_mux_select" default="0" />
  <arg name="vel_mux_select" default="1" />
  <arg name="sim_mode" default="false" />
  
  <node pkg="vel_pose_mux" type="vel_pose_mux" name="vel_pose_mux" output="screen">
    <param name="pose_mux_select" value="$(arg pose_mux_select)" />
    <param name="vel_mux_select" value="$(arg vel_mux_select)"/>
    <param name="sim_mode" value="$(arg sim_mode)"/>
  </node>
  
</launch>
